afk.lic

```diff
@@ -22,11 +22,21 @@ depart_on_death = settings.depart_on_death
 
 pause 1 while health < [health_threshold + 20, 100].min || spirit < [health_threshold + 20, 100].min
 
+justice_message_count = 0
+
 loop do
   line = script.gets?
   pause 0.05 unless line
 
   fput(%w(tdp time age).sample) if line =~ /you have been idle too long/i
+  
+  justice_message_count += 1 if line =~ /You hear the cries echo around you as everyone in the vicinity immediately gives/
+  justice_message_count += 1 if line =~ /authorities will try to bring you in for endangering the public/
+  
+  if justice_message_count > settings.afk_justice_threshold
```

Comment:
What do you think about defining `justice_threshold = settings.afk_justice_threshold` on line 22?